components/user_access/app/views/user_access/users/registrations/edit.html.erb

```diff
@@ -1,7 +1,7 @@
 <h2>Edit <%= resource_name.to_s.humanize %></h2>
 
 <%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
-  <%= render "user_registrations/shared/error_messages", resource: resource %>
+  <%= render "user_access/users/shared/error_messages", resource: resource %>
```

Comment:
Just curious, why is it `user_access/users` and not `user_access/user_registrations` ?

Reply:
it could be `user_registrations` if we want. But as I see it, the routes/controllers/views don't need to know about the `UserRegistration` model, that's why I put everything under `users` (for routes, controllers and views)